### ISSUE TITLE
Trouble Configuring Plugin Settings

### ISSUE THREAD

**Issue by EliotKhachi** [cobalamin/obsidian-remarkable#11]
Hi, thank you for developing this awesome plugin. I'm eager to try it out.

I want to make sure I'm understanding the installation instructions. I'm pretty new to obsidian so I'm not great at navigating around yet.
Is this the settings window for the Obsidian42 plugin?
![Screenshot from 2023-05-20 13-03-25](https://github.com/cobalamin/obsidian-remarkable/assets/68179450/48588edd-e558-4118-933c-0b73647858a4)

In the plugin setting configuration step, I'm not understanding what is meant by `"reSnap executable": Enter the absolute path to the previously installed reSnap.sh"`.

Where do I enter this absolute path?

**Comment by EliotKhachi** [comment]
Oh it seems like Obsidian42 is a plug-in manager.

**Comment by cobalamin** [comment]
Hi there, did it work out for you to use my plugin with BRAT? :)

**Comment by EliotKhachi** [comment]
Hey! Hope you're doing well, this is very cool stuff :)

I've got reSnap working and know how to send images from my remarkable to
my rMDrawings folder manually by doing `./reSnap.sh -o
/..../rMDrawings/image.png`.
[image: Screenshot from 2023-05-20 14-40-40.png]

Still super new to obsidian so I'm not sure what the advantage of BRAT is
or even how to use plugins. I added your plugin manually (I think),
although I'm not sure how to use it to run `reSnap`, apply the styles to
the images, and postprocess the images.
[image: Screenshot from 2023-05-20 14-39-52.png]

Eliot
